packages/ui/src/app/config.ts

```diff
@@ -0,0 +1,6 @@
+export const OLYMPIA_TESTNET_NODE_SOCKET =
+  process.env.REACT_APP_OLYMPIA_TESTNET_NODE_SOCKET || 'wss://olympia-dev.joystream.app/rpc'
+export const OLYMPIA_TESTNET_QUERY_NODE =
+  process.env.REACT_APP_OLYMPIA_TESTNET_QUERY_NODE || 'https://olympia-dev.joystream.app/query/server/graphql'
```

Comment:
Could you check if this couldn't be done differently? AFAIR `webpack.DefinePlugin` will inline those: [https://webpack.js.org/plugins/define-plugin/#usage](https://webpack.js.org/plugins/define-plugin/#usage)

This would require adding types for globals https://github.com/Joystream/pioneer/blob/7e8cdde48c329ae0a502e751fbc3d68b34ed02a8/packages/ui/src/types.d.ts#L2-L3